scripts/prepare-qa-build.js

```diff
@@ -0,0 +1,32 @@
+const shell = require('shelljs');
+const nodegit = require('nodegit');
+const {
+  gitDescribeSync
+} = require('git-describe');
+
+const gitInfo = gitDescribeSync();
+if (gitInfo.dirty) {
+  console.log('Please start with a clean repo');
+  shell.exit(1);
+}
+
+const qaVersion = gitInfo.semverString.replace('+', '.');
+
+(async() => {
```

Comment:
This causes an error on `node 7.4.0` without running this script command through `babel` first (accomplished in the other npm scripts by using `babel-register`).